docs/resources/Channel.md

```diff
@@ -1040,3 +1154,117 @@ Adds a recipient to a Group DM using their access token
 ## Group DM Remove Recipient % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/recipients/{user.id#DOCS_RESOURCES_USER/user-object}
 
 Removes a recipient from a Group DM
+
+## Start a public thread % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/threads
```

Comment:
This endpoint isn't really consistent with the other endpoints for threads. It would make more sense to merge this with the private threads endpoint i.e. `channels/channel_id/threads` and have a `private` boolean field in the json payload. And of course a `message_id` field which would be required for public threads.

This also works as a future proof for potentially being able to create public threads without needing a message to be provided.